src/atlas/interpolation/method/fe/FiniteElement.h

```diff
@@ -28,7 +28,9 @@ namespace method {
 
 class FiniteElement : public Method {
 public:
-    FiniteElement( const Config& config ) : Method( config ) {}
+    FiniteElement( const Config& config ) : Method( config ) {
+      if (config.has("maxFractionElemsToTry") ) { config.get("maxFractionElemsToTry", maxFractionElemsToTry_); }
```

Comment:
The convention is to use snake_case for the configuration keys, so "max_fraction_elems_to_try".

Reply:
should be fixed in the most recent commit.